server/src/tasks/eth_funder.js

```diff
@@ -16,16 +18,23 @@ const ethFunder = withAccount(async (account, { accountAddress }) => {
   if (networkType !== 'ropsten') {
     throw Error('Fund ETH available only for ropsten')
   }
-  await web3.eth.sendTransaction({
+  const promise = web3.eth.sendTransaction({
     to: accountAddress,
     from: account.address,
     value: toWei(bonus),
     nonce: account.nonce,
     gasPrice: '1000000000',
     gas: config.get('gasLimitForTx.funder')
+  }).on('transactionHash', (hash) => {
+    job.attrs.data.txHash = hash
+    job.save()
+  }).on('receipt', (receipt) => {
+    job.attrs.data.receipt = receipt
```

Comment:
this might be problematic, cause receipt can be a complex object

Reply:
Good point, I'll just set the receipt to true.